### PR TITLE
Increase api version for Beats monitoring

### PR DESCRIPTION
**Pull request by monicasarbu**
This PR contains two changes:
- increase the version of the API for Beats monitoring
- add the response body with the error message returned by Elasticsearch, in case of an error

### CODE REVIEW

**Comment by tsg** [comment]
@monicasarbu, can you rebase this one, please? That should fix the intake job on Jenkins.

**Comment by monicasarbu** [comment]
done @tsg

File: libbeat/outputs/elasticsearch/client.go [hunk 1]
```diff
@@ -719,16 +719,18 @@ func (conn *Connection) execHTTPRequest(req *http.Request) (int, []byte, error)
...
 	if err != nil {
 		return status, nil, retErr
 	}
+
+	if status >= 300 {
+		// add the response body with the error returned by Elasticsearch
+		retErr = fmt.Errorf("%v: %s", resp.Status, obj)
+	}
+
 	return status, obj, retErr
```
Comment (urso) [inline libbeat/outputs/elasticsearch/client.go]: how about:

```
status := resp.StatusCode
obj, err := ioutil.ReadAll(resp.Body)
if err != nil {
  return status, nil, err
}

if status >= 300 {
  err = fmt.Errorf("%v: %s", resp.Status, obj)
}

return status, obj, err
```

Comment (monicasarbu) [inline libbeat/outputs/elasticsearch/client.go]: @urso Done, thanks for the review.

**Pull request merged**